src/WordPress/JsonMapper/Evaluators/NamespaceResolver.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace WordPress\JsonMapper\Evaluators;
+
+use PhpParser\NodeTraverser;
+use PhpParser\ParserFactory;
+use WordPress\JsonMapper\Import;
+use WordPress\JsonMapper\JsonMapper;
+use WordPress\JsonMapper\ObjectWrapper;
+use WordPress\JsonMapper\Property\Property;
+use WordPress\JsonMapper\Property\PropertyMap;
+use WordPress\JsonMapper\Property\PropertyType;
+use WordPress\JsonMapper\UseNodeVisitor;
+
+class NamespaceResolver implements JsonEvaluatorInterface {
+
+	private $scalar_types = array( 'string', 'bool', 'boolean', 'int', 'integer', 'double', 'float' );
+
+	public function __construct() {}
+
+	public function evaluate(
+		\stdClass $json,
+		ObjectWrapper $object_wrapper,
+		PropertyMap $property_map,
+		JsonMapper $mapper
+	) {
+		foreach ( $this->fetchPropertyMapForObject( $object_wrapper, $property_map ) as $property ) {
+			$property_map->addProperty( $property );
+		}
+	}
+
+	private function fetchPropertyMapForObject( ObjectWrapper $object, PropertyMap $originalPropertyMap ): PropertyMap {
```

Comment:
nitpick: 
```suggestion
	private function computePropertyMapForObject( ObjectWrapper $object, PropertyMap $originalPropertyMap ): PropertyMap {
```

Reply:
Sure. Changed to `compute_property_map` since it takes an object and it is the only computing method there.